GX-TimeFrequency/src/minimax_omega.F90

```diff
@@ -0,0 +1,3079 @@
+            2.240461161972653059e+01_dp*4_dp, 3.214137896828840724e+01_dp*4_dp, 4.651372161014791828e+01_dp*4_dp, 6.822612573034811589e+01_dp*4_dp, 1.024653370494206541e+02_dp*4_dp,&
+            1.610806535928946346e+02_dp*4_dp, 2.784572085100483605e+02_dp*4_dp, 5.956911596592910882e+02_dp*4_dp, 2.267969788694746967e+03_dp*4_dp,&
+            1.480254215428695730e-01_dp, 4.579337644772357163e-01_dp, 8.110189756982952858e-01_dp, 1.241560090115072512e+00_dp, 1.792659440692006001e+00_dp, 2.521128699093249548e+00_dp,&
+            3.504086793975091485e+00_dp, 4.848189042890807876e+00_dp, 6.702769818685197833e+00_dp, 9.278757753466003066e+00_dp, 1.287611378507417648e+01_dp, 1.792391058605255694e+01_dp,&
+            2.503929472092100994e+01_dp, 3.511494788481102347e+01_dp, 4.945026357689698671e+01_dp, 6.995142494837453739e+01_dp, 9.944529593358640795e+01_dp, 1.421967669711051485e+02_dp,&
+            2.048365962244545813e+02_dp, 2.982605268262436766e+02_dp, 4.422824727194411025e+02_dp, 6.798553440442351530e+02_dp, 1.135413559310467008e+03_dp, 2.428192275071517088e+03_dp,&
+            2.983340021067058534e-01_dp*4_dp, 3.263141519727263962e-01_dp*4_dp, 3.855062739015840401e-01_dp*4_dp, 4.826798657096443845e-01_dp*4_dp, 6.288796171174622085e-01_dp*4_dp,&
+            8.407612053062157864e-01_dp*4_dp, 1.142716131984746086e+00_dp*4_dp, 1.570068541404567863e+00_dp*4_dp, 2.173800002227348482e+00_dp*4_dp, 3.027515759414763252e+00_dp*4_dp,&
+            4.237747796373835918e+00_dp*4_dp, 5.959280561489141093e+00_dp*4_dp, 8.418138358242062580e+00_dp*4_dp, 1.194649769100173309e+01_dp*4_dp, 1.703678998742747908e+01_dp*4_dp,&
+            2.442847613874887003e+01_dp*4_dp, 3.525570572372998868e+01_dp*4_dp, 5.132398148002147309e+01_dp*4_dp, 7.570555798260247116e+01_dp*4_dp, 1.142569409241104665e+02_dp*4_dp,&
+            1.802699406534468949e+02_dp*4_dp, 3.121779089126736153e+02_dp*4_dp, 6.676444661490867247e+02_dp*4_dp, 2.538283709753751737e+03_dp*4_dp,&
+            1.495207465076293374e-01_dp, 4.628547668063078824e-01_dp, 8.207591322438863024e-01_dp, 1.258748491208634190e+00_dp, 1.821669292883810742e+00_dp, 2.568904882913708221e+00_dp,&
+            3.581443179192572934e+00_dp, 4.971815820961063004e+00_dp, 6.898325411438340105e+00_dp, 9.585627327738283299e+00_dp, 1.335474040584177935e+01_dp, 1.866711361962545368e+01_dp,&
+            2.618979183629015495e+01_dp, 3.689256725772209222e+01_dp, 5.219436041282234839e+01_dp, 7.418729829085677352e+01_dp, 1.059888838572088900e+02_dp, 1.523217269704461501e+02_dp,&
+            2.205481261826462855e+02_dp, 3.227717685539633976e+02_dp, 4.809542755104906746e+02_dp, 7.425159256330131257e+02_dp, 1.244391510408988779e+03_dp, 2.667302101970366493e+03_dp,&
+            3.013957085537650737e-01_dp*4_dp, 3.302604655479730589e-01_dp*4_dp, 3.914004559789399096e-01_dp*4_dp, 4.919676878117055230e-01_dp*4_dp, 6.436
```

Comment:
This should be handled by the class methods

Reply:
Do you mean this multiplications by `4_dp`? TBH I do not know why they are there

Reply:
The multiplication by 4_dp is to make the grids consistent with those available in CP2K. After discussion with Matteo, it turns out that we may get rid of this multiplication in GreenX as Matteo needs to refactor the results at the end of calculations in ABINIT. So, we need to decide if we keep the multiplication or not.